src/solver/simulation/common-eco-adq.h

```diff
@@ -88,16 +88,16 @@ void ComputeFlowQuad(Data::Study& study,
 /*!
 ** \brief Hydro Remix
 **
-** \param study The Antares study
+** \param areas : the areas of study
 ** \param problem The weekly problem, from the solver
 ** \param hourInYear The hour in the year of the first hour in the current week
-** \param nbHours The number of hour for a week
 */
-void RemixHydroForAllAreas(const Data::Study& study,
+void RemixHydroForAllAreas(const Data::AreaList& areas,
                            PROBLEME_HEBDO& problem,
+                           Data::SheddingPolicy sheddingPolicy,
+                           Data::SimplexOptimization splxOptimization,
                            uint numSpace,
-                           uint hourInYear,
-                           uint nbHours);
+                           uint hourInYear);
```

Comment:
This is context info

Reply:
could you give some more explanations about what you mean ?

Reply:
This free function **RemixHydroForAllAreas(...)** is called from a post process command, which retrieves  **hourInYear** from **optRuntimeData**.
We could move the body of this function to the **run(...)** method of this post process command. But it would have some consequences, as  **RemixHydroForAllAreas(...)**is also called in **adequacy.cpp**. 
So I preferred postponing this refactoring.